types/set-cookie-parser/index.d.ts

```diff
@@ -27,6 +27,10 @@ declare module "set-cookie-parser" {
             httpOnly?: boolean;
         }
 
+        interface CookieMap {
+          [s: string]: Cookie;
```

Comment:
probably, using `name` instead of `s` as key name will be better for readability, because in case of `map: true` key is name of a cookie.